test/e2e/framework/test_context.go

```diff
@@ -45,6 +45,8 @@ type TestContextType struct {
 	ReportDir                string
 	ReportPrefix             string
 	Prefix                   string
+	QdrImage                 string
+	OpImage                  string
```

Comment:
Since we are adding/changing this, can we call it `OperatorImage`?

Reply:
Sure! I will rename it.